### ISSUE TITLE
Saving as XLSX doesn't open in Office 365, XLS does

### ISSUE THREAD

**Issue by SourceCode** [rainabba/jquery-table2excel#35]
This evening I gave this plug-in a run, thank you for your work.  What I noticed was a few things - first you are suggesting for people to use the dist/ files and I assume this is after a grunt build? Without the grunt build I get an error about "no MSIE".  See screenshot:
![nomsie](https://cloud.githubusercontent.com/assets/55036/13735556/5f35b82e-e968-11e5-85a1-0ad795d19c0c.PNG)

Finally I would like to provide you the two examples for my bug report.  What I notice is that if I save the files as XLS they will open (with warning) in the latest version of Office.  If I use the out of the box extension - they will not open at all and Excel just claims that are incorrectly formatted.

I will attach two examples of the output (which are the same - only difference is file extension) for your review.  

**XLSX**
[export-report.xlsx](https://github.com/rainabba/jquery-table2excel/files/171439/export-report.xlsx)

(Sorry it looks like I can not upload an XLS to github, only XLSX - just make a copy and change the extension)

Thank you again - this plug-in has saved me some time!

**Comment by rainabba** [comment]
I'm glad it helped. In short, this is a hack developed for internal use and depends on techniques that render it less than ideal for production.

tl;dr (now added to the README.md)

This plugin is a hack on a hack. The .xls extension is the only way [some versions] of excel will even open it, and you will get a warning about the contents which can be ignored. The plugin was developed against Chrome and other have contributed code that should allow it to work in Firefox and Safari, but inconsistently since it's a hack that's not well supported anywhere but Chrome. I would not use this in public production personally and it was developed for an Intranet application where users are on Chrome and had known versions of Excel installed and the users were educated about the warning. These users also save-as in Excel so that when the files are distributed, the end-users don't get the warning message.

**Comment by SourceCode** [comment]
Ok that makes total sense.  Thanks for letting me know.  A question for you - would you care if I fork this and update it so that it does indeed work across platforms? I reviewed the code and noticed that I could improve it and cover the scenarios possibly.  I know the warnings will always be in place.

Thank you again for taking the time to respond and inform me on the situation.

**Comment by rainabba** [comment]
Forks and PRs are very welcome. That's the beauty of OSS; people like me start an idea and everyone else that wants to see it improved can contribute and GitHub ensures that everyone gets due credit for those contributions. :)

This is a nice tutorial about how to use GitHub Pull Requests (PRs): https://yangsu.github.io/pull-request-tutorial/
